builtin/credential/cert/backend_test.go

```diff
@@ -19,21 +34,26 @@ func TestBackend_basic_CA(t *testing.T) {
 	if err != nil {
 		t.Fatalf("err: %v", err)
 	}
-	b, err := Factory(&logical.BackendConfig{
-		System: &logical.StaticSystemView{
-			DefaultLeaseTTLVal: 300 * time.Second,
-			MaxLeaseTTLVal:     1800 * time.Second,
-		},
-	})
+	crl, err := ioutil.ReadFile("../../../test/ca/root.crl")
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
 	logicaltest.Test(t, logicaltest.TestCase{
-		Backend: b,
+		Backend: testFactory(t),
 		Steps: []logicaltest.TestStep{
 			testAccStepCert(t, "web", ca, "foo"),
 			testAccStepLogin(t, connState),
+			testAccStepCertLease(t, "web", ca, "foo"),
 			testAccStepCertTTL(t, "web", ca, "foo"),
 			testAccStepLogin(t, connState),
 			testAccStepCertNoLease(t, "web", ca, "foo"),
 			testAccStepLoginDefaultLease(t, connState),
+			testAccStepAddCRL(t, crl, connState),
```

Comment:
I wonder if we should copy/paste and have a new _CRL test, since this thing is getting to be a bit of a beast